pkg/controllers/networkpolicy/networkpolicy.go

```diff
@@ -1397,6 +1393,7 @@ func (n *networkPolicy) update(anp *antreanetworking.NetworkPolicy, recompute bo
 	modifiedAddr := make([]string, 0)
 	if recompute {
 		addedAddr, _ = diffAddressGrp(nil, anp.Rules)
+		r.Log.V(1).Info("Compute rules", "networkPolicy", n.Name)
```

Comment:
too many recompute logs.. 1396, 1418, 1440. Just have logs in computeRules func only..

Reply:
These logs makes it easier to understand np controller workflow.